cmd/flow-rosetta-server/main.go

```diff
@@ -111,9 +112,12 @@ func run() int {
 	server.HideBanner = true
 	server.HidePort = true
 	server.Use(middleware.Logger())
+	server.POST("/network/list", ctrl.Networks)
+	server.POST("/network/options", ctrl.Options)
+	server.POST("/network/status", ctrl.Status)
 	server.POST("/account/balance", ctrl.Balance)
```

Comment:
Will we support the `/account/coins` endpoint?

Reply:
No, it doesn't exist on Flow.